__tests__/app.test.js

```diff
@@ -257,35 +257,31 @@ describe("POST /api/articles/:article_id/comments", () => {
     })
 })
 
-// describe("DELETE /api/comments/:comment_id", () => {
-//     test("responds with status 204 and no content" , () => {
-//         return request(app).delete('/api/comments/1')
-//         .expect(204)
-//         .then(({body}) => {
-//             expect(body).toEqual({})
-//         })
-//     })
-//     test("returns 404 error if comment_id doesnt exist", () => {
-//         return request(app).post('/api/comments/1203')
-//         .expect(404)
-//         .then(({ body }) => {
-//             expect(body.msg).toBe(`comment_id with value 1203 does not exist in comments`)   
-
-//         })
-//     })
-
-// })
-
+describe("DELETE /api/comments/:comment_id", () => {
+    test("responds with status 204 and no content" , () => {
+        return request(app).delete('/api/comments/1')
+        .expect(204)
+        .then(({body}) => {
```

Comment:
Here the body is not coming from your API, but from Supertest, so don't need to include it.